### PR TITLE
updates to diarization docs

### PR DESCRIPTION
**Pull request by SandraRodgers**

### CODE REVIEW

File: developers/src/pages/documentation/features/diarize/index.mdx [hunk 1]
```diff
@@ -20,8 +21,27 @@ import CodeGroup from '../../../../shared/components/code/CodeGroup.astro';
...
+[Speaker:0] I'm doing well. Thank you. May I please have your name?
+[Speaker:1] My name is Blake...
+```
+
 To learn more about diarization and multichannel audio, and to learn when to use Deepgram's Diarization or Multichannel feature, see [Understanding when to Use the Multichannel and Diarization Features](/documentation/guides/multichannel-vs-diarization/).
 
+<Alert type="info">
+**UPDATE**
+<br/>
+Deepgram's newly released version of our Speaker Diarization technology provides better accuracy and performance. Its advanced speaker separation accurately identifies speakers in complex audio streams, reducing errors where two speakers are identified as one. Additionaly, the new diarizer can identify and count speakers more accurately, reducing instances where one speaker is split between two labels. The result is more readable transcripts.
```
Comment (incubatize) [inline developers/src/pages/documentation/features/diarize/index.mdx:37]: See me comment below.

File: developers/src/pages/documentation/features/diarize/index.mdx [hunk 2]
```diff
@@ -36,9 +56,10 @@ To enable Diarization, when you call Deepgram’s API, add a `diarize` parameter
 
 <Alert type="warning">
   
-If you want to use our legacy Diarization feature, when you call Deepgram’s API, add an additional `diarize_version` parameter set to `2021-07-14.0` in the query string:
-
-`diarize=true&diarize_version=2021-07-14.0`
+With the release of improved diarization, we will be deprecating the `diarize_version` parameter and will be retiring the old diarizer on April 3, 2023.
```
Comment (incubatize) [inline developers/src/pages/documentation/features/diarize/index.mdx:59]: Can we add the above para here?
 "We've recently released an improved version of Diarization. Its advanced speaker separation accurately identifies speakers in complex audio streams, reducing errors where two speakers are identified as one. Additionaly, the new Diarizer can identify and count speakers more accurately, reducing instances where one speaker is split between two labels."

File: developers/src/pages/documentation/features/diarize/index.mdx [hunk 3]
```diff
@@ -36,9 +56,10 @@ To enable Diarization, when you call Deepgram’s API, add a `diarize` parameter
 
 <Alert type="warning">
   
-If you want to use our legacy Diarization feature, when you call Deepgram’s API, add an additional `diarize_version` parameter set to `2021-07-14.0` in the query string:
-
-`diarize=true&diarize_version=2021-07-14.0`
+With the release of improved diarization, we will be deprecating the `diarize_version` parameter and will be retiring the old diarizer on April 3, 2023.
+<br/>
+Currently, you can call the old diarizer using the below URL:
+`https://api.deepgram.com/v1/listen?tier=enhanced&diarize=true&diarize_version=2021-07-14.0`
```
Comment (incubatize) [inline developers/src/pages/documentation/features/diarize/index.mdx:63]: Can we add below para after the last line above?

To access the latest diarizer, all you need to do is add **diarize=true** to your URL: `https://api.deepgram.com/v1/listen?diarize=true`

 We encourage you to switch to the improved Diarizer as soon as possible to ensure that you are taking advantage of the latest advancements in our technology.

**Review by incubatize — COMMENTED** [review]
added my comment. thanks!

File: developers/src/pages/documentation/features/diarize/index.mdx (hunk 1 above)
Comment (SandraRodgers) [reply developers/src/pages/documentation/features/diarize/index.mdx:37]: Are you saying that this alert paragraph should be removed?

Comment (incubatize) [reply developers/src/pages/documentation/features/diarize/index.mdx]: @SandraRodgers -looks good.

File: developers/src/pages/documentation/features/diarize/index.mdx [hunk 4]
```diff
@@ -36,9 +37,16 @@ To enable Diarization, when you call Deepgram’s API, add a `diarize` parameter
 
 <Alert type="warning">
   
-If you want to use our legacy Diarization feature, when you call Deepgram’s API, add an additional `diarize_version` parameter set to `2021-07-14.0` in the query string:
+We've recently released an improved version of Diarization. Its advanced speaker separation accurately identifies speakers in complex audio streams, reducing errors where two speakers are identified as one. Additionally, the new diarizer can identify and count speakers more accurately, reducing instances where one speaker is split between two labels. The result is more readable transcripts.
```
Comment (incubatize) [inline developers/src/pages/documentation/features/diarize/index.mdx:40]: can we update this to 
We've recently released an improved version of Diarization (PRE-RECORDED Only).

**Review by incubatize — COMMENTED** [review]
@SandraRodgers  - added a comment. Can you update?

**Pull request merged**